packages/xsnap/test/install.test.js

```diff
@@ -0,0 +1,20 @@
+// building xsnap locally and installing from npm follow different paths
+// through build.js, since installation targets are not obliged to have a git
+// toolchain.
+// This test verifies the post-publish experience.
+
+import test from 'ava';
+import { spawn } from 'child_process';
+import { makeCLI } from '../src/build.js';
+
+test('pack and install xsnap', async t => {
+  const npm = makeCLI('npm', { spawn });
+  const tar = makeCLI('tar', { spawn });
+  const npmRes = await npm.pipe(['pack', '--json']);
+  const {
+    0: { filename },
+  } = JSON.parse(npmRes);
+  await tar.run(['xf', filename]);
+  await npm.run(['install'], { cwd: 'package' });
```

Comment:
Rather than extracting `makeCLI`, I recommend following userland's use of [execa](https://www.npmjs.com/package/execa) for testing as seen in e.g. [fast-usdc](https://github.com/Agoric/agoric-sdk/blob/master/packages/fast-usdc/test/cli/cli.test.ts).
```suggestion
  const { stdout: npmRes } = await execa`npm pack --json`;
  const {
    0: { filename },
  } = JSON.parse(npmRes);
  await execa`tar xf ${filename}`;
  await execa({ cwd: 'package' })`npm install`;
```